docker/entrypoint.sh

```diff
@@ -39,7 +39,7 @@ else
 
 
     # Load *.troy files
-    TROYS=$(find ${CFG_DIR} -name '*.troy' -print 2>/dev/null | sort)
+    TROYS=$(find /etc/tremor/config/ -name '*.troy' | egrep -v '\.\.[0-9]{4}_[0-9]{2}_[0-9]{2}_[0-9]{2}_[0-9]{2}_[0-9]{2}' 2>/dev/null | sort)
```

Comment:
lets make this ignore all hidden files:
```suggestion
    TROYS=$(find /etc/tremor/config/ -not -path '*/.*' -type f -name '*.troy' -print 2>/dev/null | sort)
```

Reply:
👍 This is cleaner and easier to document as well. @ekarlso would the above suggestion work in your case? If so we can test in the next release candidate...

Reply:
Actually that doesnt fly, the "files" in kubernetes are symlinks rather :) just try it in one of any running it in a pod and you'll see :)